packages/data-sql/src/types.ts

```diff
@@ -48,9 +50,15 @@ export interface AbstractSqlQuery {
 	from: string;
 	limit?: ParameterIndex;
 	offset?: ParameterIndex;
+	order?: AbstractSqlOrder[];
 	parameters: (string | boolean | number)[];
 }
 
+export type AbstractSqlOrder = {
+	orderBy: AbstractQueryNodeSortTargets;
```

Comment:
Has it already been considered that the target could be processed at this level, or does it clearly make more sense to do that in the individual adapters?

Reply:
I spoke with Rijk about the relational nodes. result: for now we concentrate on the basics for everything, but do  relational things later on. that would make primitive and function nodes the only thing here to consider atm

Reply:
Okay, thanks for clarifying! Sounds like a good plan 🚀